azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -72,12 +85,12 @@ def run(self, circuit, **kwargs):
             if opt in input_params:
                 input_params[opt] = kwargs.pop(opt)
 
-        logger.info(f"Submitting new job for backend {self.name()}")
+        logger.info(f"Submitting new job for basckend {self.name()}")
         job = AzureQuantumJob(
             backend=self,
             name=circuit.name,
             target=self.name(),
-            input_data=input_data,
+            input_data=json.dumps(input_data),
```

Comment:
no, this is wrong. the input_data is an object, not a string.

Reply:
Yes you're right, it's a `bytes` object. I've corrected this and am using `IonQ._encode_input_data`.